src/crypto/test/crypto.cpp

```diff
@@ -1109,4 +1215,44 @@ TEST_CASE("Sign and verify with RSA key")
       mdtype,
       verify_salt_legth));
   }
-}
\ No newline at end of file
+}
+
+TEST_CASE("COSE sign & verify")
+{
+  std::shared_ptr<KeyPair_OpenSSL> kp =
+    std::dynamic_pointer_cast<KeyPair_OpenSSL>(
+      ccf::crypto::make_key_pair(CurveID::SECP384R1));
+
+  std::vector<uint8_t> payload{1, 10, 42, 43, 44, 45, 100};
+  const std::unordered_map<int64_t, std::string> protected_headers = {
+    {36, "thirsty six"}, {47, "hungry seven"}};
+  auto cose_sign = cose_sign1(*kp, protected_headers, payload);
+
+  if constexpr (false) // enable to see the whole cose_sign as byte string
```

Comment:
maybe use NDEBUG, so this is shown in debug builds?

Reply:
It's not really for debug/release, it's more of being able to easily print it to verify manually only if you need it.

I could've removed that code after debugging, but I considered that to be useful, because it's not a trivial thing to correctly print raw payload as bytes if you didn't do it before (like me this time).